tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-integration-test.php

```
@@ -264,6 +288,129 @@ public function test_is_merchant_enabled_returns_false_when_disabled() {
+			update_option( $this->last_upload_option, $cached_record, false );
+		}
+		if ( ! $filter_enabled ) {
+			add_filter( 'wc_stripe_agentic_commerce_feed_dedupe_enabled', '__return_false' );
+		}
+
+		$integration  = new \WC_Stripe_Agentic_Commerce_Integration();
+		$is_unchanged = new \ReflectionMethod( \WC_Stripe_Agentic_Commerce_Integration::class, 'is_feed_unchanged' );
+		$is_unchanged->setAccessible( true );
+
+		$this->assertSame( $expected, $is_unchanged->invoke( $integration, $candidate_hash ) );
+	}
```

PHPMD (2.15.0)

[error] The method test_is_feed_unchanged_scenarios is not named in camelCase. (undefined)

(CamelCaseMethodName)